documentation/api/io.strimzi.api.kafka.model.kafka.tieredstorage.TieredStorageCustom.adoc

```diff
@@ -1,9 +1,13 @@
-To configure the tiered storage feature, set the `type` property to `custom`.
+Enables custom tiered storage for Kafka.
 
-`RemoteStorageManager` is a Kafka interface for managing the interaction between Kafka and remote tiered storage solutions.
-Custom tiered storage enables the use of a custom `RemoteStorageManager` configuration.
-If custom tiered storage is enabled, Strimzi uses the link:https://github.com/apache/kafka/blob/trunk/storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManager.java[`TopicBasedRemoteLogMetadataManager`] for Remote Log Metadata Management (RLMM) configuration.
-If you want to use custom tiered storage, you must first add the tiered storage plugin to the Strimzi image by building a custom container image.
+If you want to use custom tiered storage, you must first add a tiered storage for Kafka plugin to the Strimzi image by building a custom container image.
+
+Custom tiered storage configuration enables the use of a custom `RemoteStorageManager` configuration.
+`RemoteStorageManager` is a Kafka interface for managing the interaction between Kafka and remote tiered storage.
+
+If custom tiered storage is enabled, Strimzi uses the link:https://github.com/apache/kafka/blob/trunk/storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManager.java[`TopicBasedRemoteLogMetadataManager`] for Remote Log Metadata Management (RLMM).
+
+NOTE: Tiered storage is available a technology preview with Strimzi, so it is not suitable for production environments.
```

Comment:
We do not have any technology preview and do not consider this technology preview. Kafka considers this currently early access with several limitations. That is what we should warn here about.

Reply:
I've made it a warning and linked to the limitations

> WARNING: Tiered storage is an early access Kafka feature, which is also available in Strimzi. 
> Due to its https://kafka.apache.org/documentation/#tiered_storage_limitation[current limitations^], it is not recommended for production environments.